src/broker/runat.c

```diff
@@ -465,9 +465,9 @@ int runat_push_shell_command (struct runat *r,
 
     /*   For shell commands run the target cmdline in a separate process
```

Comment:
commit message for "always execute rc2 in a separate process group": s/doe/done/